instrumentation/vertx-reactive-3.5/src/test/groovy/VertxReactivePropagationTest.groovy

```diff
@@ -68,7 +69,8 @@ class VertxReactivePropagationTest extends AgentTestRunner {
           parent()
           attributes {
             "${SemanticAttributes.NET_PEER_PORT.key()}" Long
-            "${SemanticAttributes.NET_PEER_IP.key()}" { it == null || it == "127.0.0.1" } // Optional
+            "${SemanticAttributes.NET_PEER_IP.key()}" { it == null || it == "127.0.0.1" }
+            // Optional
```

Comment:
this happens when Intellij wraps our end-of-line comments. usually i move the eol comment above the line, but in this case i don't think the comment is very helpful, so suggest we just delete it
```suggestion
```